src/licensedcode/data/rules/lgpl-2.0-plus_with_libtool-exception-2.0_1.RULE

```diff
@@ -0,0 +1,20 @@
+GNU Libltdl is free software; you can redistribute it and/or
```

Comment:
It would be interesting to also have another rule (and may be have only that rule) WITHOUT the libltdl reference?
For instance rather than start with:
`GNU Libltdl is free software; you can redistribute it and/or`
use this:
`GNU is free software; you can redistribute it and/or` 

and remove the libtdl word elsewhere?
Because of the way the license detection ignores words it doe not know about (such as libtdl) this will not have any impact and will be detected with the automaton... And it can also detected exactly several other variants.

Reply:
This works perfectly for this case, i.e. (having only that rule) it would also detect if the text's exactly same apart from "Libltdl". I was wondering if I can keep the original rule with a higher relevance, and the non-liblltdl one with a lower relevance score, so it detects other similar stuff but in case of multiple matches would prioritize other more relevant matches. I'm pushing that.

Reply:
IMHO I think there's some work to be done in standardizing `relevance scores` and `minimum coverages` across all the rules, after performing statistics on false positive/unknown matches, as these are kind of happening regularly in bugs. This would be one of the smaller important tasks of the GSoC project, what do you think?

Reply:
Is there some doc on how `relevance scores` are given out to rules? I see the general trend in rule names/texts vs their relevance/relevance scores, and how they affect license scores, but are there any strict guidelines I mean?

Reply:
> liblltdl

IMHO only the one without liblltdl would be enough

Reply:
> IMHO I think there's some work to be done in standardizing `relevance scores` and `minimum coverages` across all the rules, after performing statistics on false positive/unknown matches, as these are kind of happening regularly in bugs. This would be one of the smaller important tasks of the GSoC project, what do you think?

excellent idea :+1:

Reply:
> Is there some doc on how `relevance scores` are given out to rules? I see the general trend in rule names/texts vs their relevance/relevance scores, and how they affect license scores, but are there any strict guidelines I mean?

The relevance is computed dynamically based on the size of a rule OR stored and assigned manually as a curated value based on judgment.
See 
- https://github.com/nexB/scancode-toolkit/blob/d18fd2db7ba2dd893a4c46bdbbc9d7db8fb710c1/src/licensedcode/models.py#L729
- https://github.com/nexB/scancode-toolkit/blob/d18fd2db7ba2dd893a4c46bdbbc9d7db8fb710c1/src/licensedcode/models.py#L1089

Reply:
> IMHO only the one without liblltdl would be enough

Done.